monkeytype/stubs.py

```diff
@@ -335,20 +335,29 @@ def render_annotation(anno: Any) -> str:
     if _is_optional(anno):
         elem_type = _get_optional_elem(anno)
         rendered = render_annotation(elem_type)
```

Comment:
nit: let's not reuse the variable name `rendered` for two different things here, let's just inline this into the next line

Reply:
Fixed.